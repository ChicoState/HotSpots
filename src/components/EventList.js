import React from 'react';
import { View, Text, StyleSheet, FlatList } from 'react-native';


const EventList = () => {
    return (
        <View>

        </View>
    )
}

const styles = StyleSheet.create({});

export default EventList;