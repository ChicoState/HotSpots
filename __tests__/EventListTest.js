import "react-native";
import React from "react";
import DealScreen from "../src/screens/DealsScreen";
import EventList from "../src/components/EventList";
import renderer from "react-test-renderer";
import act from "react-test-renderer";
import TestRenderer from "react-test-renderer";

describe("render tests", () => {
  jest.useFakeTimers();
  it("eventlist", async done => {
    var snap;
    await TestRenderer.act(async () => {
      snap = renderer.create(<EventList />);
    });
    let tree = snap.toJSON;
    await expect(tree).toMatchSnapshot();
    done();
  });
});
