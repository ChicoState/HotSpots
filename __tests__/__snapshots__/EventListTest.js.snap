// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render tests eventlist 1`] = `[Function]`;
